### PR TITLE
Add call indicator to participant list

### LINKED ISSUE #2558: Call indicator missing in participant list
There used to be a camera icon in the participant list when a used joined the call.

We might even use different icons for:
* Joined
* Joined + Audio
* Joined + Video
* Joined + Screenshare

### PR DESCRIPTION
**Pull request by nickvergessen**
Bildschirmfoto von 2020-01-23 12-46-03

Fix #2558

### CODE REVIEW

File: src/components/RightSidebar/Participants/ParticipantsList/Participant/Participant.vue [hunk 1]
```diff
@@ -144,6 +145,16 @@ export default {
 			// source: groups, circles
 			return 'icon-contacts'
 		},
+		callIconClass() {
+			if (this.participant.inCall === PARTICIPANT.CALL_FLAG.DISCONNECTED) {
+				return ''
+			}
+			const hasVideo = this.participant.inCall & PARTICIPANT.CALL_FLAG.WITH_VIDEO
```
Comment (ma12-co) [inline src/components/RightSidebar/Participants/ParticipantsList/Participant/Participant.vue]:
```suggestion
			const hasVideo = this.participant.inCall && PARTICIPANT.CALL_FLAG.WITH_VIDEO
```
That was always evaluating true
Screenshot_20200123_130511

File: src/components/RightSidebar/Participants/ParticipantsList/Participant/Participant.vue [hunk 2]
```diff
@@ -144,6 +145,16 @@ export default {
 			// source: groups, circles
 			return 'icon-contacts'
 		},
+		callIconClass() {
+			if (this.participant.inCall === PARTICIPANT.CALL_FLAG.DISCONNECTED) {
```
Comment (ma12-co) [inline src/components/RightSidebar/Participants/ParticipantsList/Participant/Participant.vue]: I think this is not working

Screenshot_20200123_130511

File: src/components/RightSidebar/Participants/ParticipantsList/Participant/Participant.vue (hunk 1 above)
Comment (nickvergessen) [reply src/components/RightSidebar/Participants/ParticipantsList/Participant/Participant.vue]: No, we want to check if the binary is set.

File: src/components/RightSidebar/Participants/ParticipantsList/Participant/Participant.vue (hunk 2 above)
Comment (nickvergessen) [reply src/components/RightSidebar/Participants/ParticipantsList/Participant/Participant.vue]: Ah, didn't check for adding participants

**Pull request merged**